source/Roadmap/Roadmap.md

```diff
@@ -0,0 +1,19 @@
+# Roadmap
+
+This page describes planned work for the ROS 2 Real-Time Working Group.
+
+
+| **Task**                                                     | **Size** | **Owner** | **Expected Completion** |
+| ------------------------------------------------------------ | -------- | --------- | ----------------------- |
+| Documentation: Complete Linux real-time guide                | Small    |           | 2nd quarter 2021        |
+| Documentation: Add instruction, motivation and <br/> goals documents | Small    |           | 2nd quarter 2021        |
+| Documentation: How to configure and run a ROS2 <br/> Real-Time application | Small    |           | 2nd quarter 2021        |
+| Documentation: Real-Time Buildfarm                           | Small    |           | 2nd quarter 2021        |
```

Comment:
is it a tutorial or howto? What kind of document is it? Probably we can provide a more detailed description?

Reply:
I split this in two tasks